plugindocs.rb

```diff
@@ -3,11 +3,11 @@
 require "fileutils"
 require "time"
 require "yaml"
-require "net/http"
 require "stud/try"
 require "pmap" # Enumerable#peach
 
 require_relative 'lib/logstash-docket'
+require_relative 'lib/logstash-docket/util/alias_plugin_util'
```

Comment:
Nitpick: can we move this require into `lib/logstash-docket`, which is supposed to load all of the `LogstashDocket` dependency?

Reply:
Moved!